dev-java/batik/files/batik-squiggle.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Squiggle
+Comment=SVG browser
+Exec=batik-squiggle
```

Comment:
Is this desktop entry supposed to show up in the applications menu?  I cannot find it after merging the ebuild

Reply:
@Leo3418 
Thanks for your help. `/usr/share/${PN}/etc/batik.policy` finally fixes it.
And I also added the slot to the launcher.